### ISSUE TITLE
Begin acquisition process for TLS certificate(s)

### ISSUE THREAD

**Issue by ElijahLynn** [department-of-veterans-affairs/va.gov-cms#3226]
AC:
- TLS Certificate request for following domains has been submitted, request is for a single certificate:

**Common Name (CN)**

- tugboat.cms.va.gov

**Subject Alternative Names (SAN)**

**DEV/STAGING/PROD**

- *.demo.cms.va.gov
- *.ci.cms.va.gov
- *.tugboat.cms.va.gov

**PROD**

- demo.cms.va.gov
- ci.cms.va.gov
- tugboat.cms.va.gov
- *.prod.demo.cms.va.gov
- *.prod.ci.cms.va.gov
- *.prod.tugboat.cms.va.gov

**STAGING**

- *.staging.demo.cms.va.gov
- *.staging.ci.cms.va.gov
- *.staging.tugboat.cms.va.gov

**DEV**

- *.dev.demo.cms.va.gov
- *.dev.ci.cms.va.gov
- *.dev.tugboat.cms.va.gov

Notes: 
If escalation is needed or a case needs to be made, we need to emphasize that all these domains are pointing towards a single server and are a single service. Well, DEV, STAGING, PROD will be separate servers and would still be nice to have the same cert.

**Comment by olivereri** [comment]
https://www.thesslstore.com/blog/security-changes-in-chrome-58/

**Comment by ElijahLynn** [comment]
> https://www.thesslstore.com/blog/security-changes-in-chrome-58/

Thank you!

Knowing the Common Name (CN) is completely ignored and the SAN is all that matters now is news to me, TIL.

**Comment by ElijahLynn** [comment]
@olivereri Looks like the spec supports mixing wildcard and non-wildcard SANs but not all CAs support them the same. So, I think we should try to get a single cert, with no common name (per your comment, is that right?) and two SAN entries.
https://stackoverflow.com/a/21496451/292408

**Comment by ElijahLynn** [comment]
Tugboat does not NEED TLS but SHOULD have it. So we are not blocked by this.

**Comment by ElijahLynn** [comment]
Eric found the request process has changed. And this is not a blocker anymore for Tugboat implementation so we are moving to sprint 19 (next).

**Comment by ElijahLynn** [comment]
We just chatted with Vikram from the PKI team and we are allowed to submit a request for an internal wildcard TLS right now. However, there is a memorandum coming that will not allow this and we will need a POAM and approval at that time to continue using the cert.

 They also mentioned we can automate this using a REST endpoint but that would require everytime an environment be base to get a new certificate, add it to Amazon Certificate Manager, attach it to the ALB and then install it on the server, which would add complexity and multiple points of failure, so we will still likely seek approval with a POAM when the time comes. 

Vikram also gave us instructions on how to submit this new request until we get access to this new system.
